### ISSUE TITLE
'confluence_editor' setting not respected when used with 'confluence_full_width'

### ISSUE THREAD

**Issue by mason3k** [sphinx-contrib/confluencebuilder#814]
Using version confluencebuilder 2.2.1:

When using confluence_editor = 'v2', my pages are still being published using the legacy editor. When I manually convert the pages further updates retain the v2 structure. I don't think it is related to the content since some of my pages are very simple (a title and a table of contents) and they are still published as v1.

I did confirm in debugging that `ConfluencePublisher`'s editor attribute is initialized to 'v2' as expected.

Happy to provide further information!

**Comment by mason3k** [comment]
I think I figured this out. If 'confluence_full_width' is set, we are overriding the 
entire [metadata][properties] dict when we're building our request:

publisher.py
```python
        if self.editor:
            page['metadata']['properties'] = {
                'editor': {
                    'value': self.editor,
                },
            }

        if self.config.confluence_full_width is not None:
            if self.config.confluence_full_width:
                content_appearance = 'full-width'
            else:
                content_appearance = 'default'
            
            # here we are wiping out the change we made above setting the editor version, because we are setting the entire properties dictionary instead of just the 'content-appearance-published' key
            page['metadata']['properties'] = {
                'content-appearance-published': {
                    'value': content_appearance,
                },
            }
```
I can make a PR if that would be useful

**Comment by mason3k** [comment]
https://github.com/sphinx-contrib/confluencebuilder/pull/815

**Comment by jdknight** [comment]
@mason3k, thanks for the report, investigation and fix! Sorry for missing that.

Changes have been merged in and will be made available next release (v2.2).

**Comment by jdknight** [comment]
v2.2 is now available on PyPI -- marking as closed.
